tasks/main.yml

```diff
@@ -7,6 +7,11 @@
 - name: Include OS variables
   include_vars: "{{ ansible_os_family }}.yml"
 
+- name: Set nomad_servers to empty list rather than the default macro if we are using consul
```

Comment:
Thx for adding the output and info! Hadn't thought about that use-case.

Not sure it's a better solution, but another option might be to update the default value macro so it evaluates to `[ ]` when `nomad_use_consul = true`.